index.html

```diff
@@ -227,5 +235,29 @@ <h1>Skills</h1>
             </div>
         </div>
     </section>
+
+    <footer class="contact">
+        <div>
+            <h5>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h5>
+            <br>
+            <form action="https://formspree.io/f/xrgvqloo" method="POST">
+                <input type="text" name="tect" maxlength="30" placeholder="Full name" required>
+                <input type="email" name="email" placeholder="Email adress" required>
+                <textarea maxlength="500" name="message" placeholder="Enter text here" required></textarea>
+                <button type="submit" name="submit">Get in touch</button>
```

Comment:
- [ ] Kindly follow the style for the `Get in touch` button as requested by the previous reviewer.
![image](https://user-images.githubusercontent.com/24703171/220432198-f31f6adb-13df-4f8d-b44f-81fdb4bddaf4.png)
When the button is active button looks like this:
![image](https://user-images.githubusercontent.com/24703171/220432474-700782f8-b86f-41ee-94de-a3e04e6c79b1.png)
When you hover with a mouse inside it button looks like this:
![image](https://user-images.githubusercontent.com/24703171/220432692-7cdff143-cb55-4899-96f0-ec2fb3b847fd.png)
When you press and hold the button looks like this:
![image](https://user-images.githubusercontent.com/24703171/220432845-03ddb098-afe6-443f-8da7-6a41f10b2c6f.png)
### Hint:
- You can use [Pseudo classes](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes).